src/main/ruby/truffleruby/core/random.rb

```diff
@@ -66,10 +66,24 @@ def random(limit)
   end
 
   def random_range(limit)
```

Comment:
I'd suggest to rename the argument to `range`, since `limit` seems a confusing name for a Range (a range has kind of 2 limits).